package.json

```diff
@@ -12,7 +12,7 @@
     "test:ends": "./sh/test-ends.sh",
     "test:lint": "./sh/lint.sh",
     "test:client-unit": "./node_modules/.bin/karma start --single-run --no-auto-watch --concurrency 1 karma.conf.js",
-    "test:server-unit": "echo 'Not yet implemented.'",
+    "test:server-unit": "./node_modules/.bin/mocha --recursive test/server-unit && set NODE_ENV=development node server/app.js",
```

Comment:
Do you need the `&& set NODE_ENV=development node server/app.js` part of this?  Unit tests usually do not need an entire server.

Reply:
Also, could you add this command to the `./sh/test.sh` command?  It should come after the "lint tests" and before the "integration tests".